### ISSUE TITLE
WHY add before bn?

### ISSUE THREAD

**Issue by howardchina** [hhb072/IntroVAE#1]
on https://github.com/hhb072/IntroVAE/blob/master/networks.py
line 40
`output = self.relu2(self.bn2(torch.add(output,identity_data)))`
I see the normal ResBlock code like this form:
`output = self.relu2(torch.add(self.bn2(output),identity_data))`

Why Add before batchNorm?

**Comment by klory** [comment]
yeah, I have the same question, it is different from https://github.com/yunjey/pytorch-tutorial/blob/56bba936197f46968aa2cc04c8096055de7f710c/tutorials/02-intermediate/deep_residual_network/main.py#L63
